contentful/export-processor/content-types/section.js

```diff
@@ -30,29 +30,29 @@ export class Section {
 
     this.paths = this.getAllPaths(this.questions[0]).map((path) => {
       const userJourney = new UserJourney(path, this);
-      userJourney.setRecommendation(recommendation);
-
+        userJourney.setRecommendation(recommendation, path);
```

Comment:
Forgive me but this gotta be aligned 😄 
```suggestion
      userJourney.setRecommendation(recommendation, path);
```